internal/report/report_operand_templates.go

```diff
@@ -0,0 +1,33 @@
+package report
+
+const (
+	operandTextReportTemplate = `
+{{ with $dot := . }}
+{{ range $index, $value := .CustomResources }}
+
+Operand Install Report
+-----------------------------------------
+Report Date: {{ now }}
+OpenShift Version: {{ $dot.OcpVersion }}
+Package Name: {{ $dot.Subscription.Package }}
+Operand Kind: {{ kind $value }}
+Operand Name: {{ name $value }}
+Operand Creation: {{ if gt $dot.OperandCount 0 }}Succeeded{{ else }}Failed{{ end }}
+-----------------------------------------
+{{ else }}
+No custom resources
+{{ end }}
+{{ end }}
+`
+
+	operandJsonReportTemplate = `
```

Comment:
This would put it on multiple lines right? This would break upload.

Reply:
I just noticed looking back that this is on 3 lines in the current code base actually, see https://github.com/opdev/opcap/blob/main/internal/report/report.go#L49

have we had issues with this?

Reply:
@bcrochet either way, I just reverted them/changed them both back to single lines

Reply:
Even if it was on three lines, the other two would be empty. Which I don't think would break the parsing. But having it split on multiple lines for one entry would definitely do so.